include/SFML/Graphics/Image.hpp

```diff
@@ -156,7 +156,7 @@ class SFML_GRAPHICS_API Image
     /// \see create, loadFromFile, loadFromMemory, saveToFile
     ///
     ////////////////////////////////////////////////////////////
-    [[nodiscard]] bool saveToMemory(std::vector<std::uint8_t>& output, const std::string& format) const;
```

Comment:
Should we pass `string_view` by value instead of reference?

Reply:
Yes, will change.